android/app/src/main/cpp/peer_audio_manager.cpp

```
@@ -1,287 +1,445 @@
     LOGI("Mixer tick loop started");
 
-    constexpr int kTickIntervalMs = 20;  // 20 ms ticks
-    constexpr int kFrameSize = 320;      // 20 ms at 16 kHz
+    constexpr int kFrameSize = audio_config::kCodecFrameSize;
+
+    // Attach this thread to the JVM once. sendAudioToPeer fires every tick
+    // per peer; per-call AttachCurrentThread / DetachCurrentThread on a
+    // 50 Hz × N-peer cadence is wasteful and adds jitter to the mixer loop.
+    // We attach at thread start, hand the same JNIEnv to sendAudioToPeer,
+    // and detach exactly once on loop exit.
+    JNIEnv* env = nullptr;
```

Clang (14.0.6)

[warning] variable 'env' is not initialized

(cppcoreguidelines-init-variables)